### PR TITLE
add appliation specific meta data to ocf read/write

### PR DESCRIPTION
**Pull request by kklipsch**

### CODE REVIEW

File: ocf_test.go [hunk 1]
```diff
@@ -11,21 +11,29 @@ package goavro_test
 
 import (
 	"bytes"
+	"fmt"
 	"testing"
 
-	"github.com/linkedin/goavro"
+	goavro "github.com/MediaMath/goavro.v2"
```
Comment (jtwatson) [inline ocf_test.go]: It seems that this import change should not be included in this PR.

**Comment by karrick** [comment]
Awesome idea to add this feature supported by the Avro spec but not in the code yet. Thanks!
